exercise/admin.py

```diff
@@ -94,17 +94,45 @@ def has_change_permission(self, request, obj=None):
         return super().has_change_permission(request, obj)
 
 
+class ExerciseEmailPropertiesListFilter(admin.SimpleListFilter):
+    title = "Exercise"
+    parameter_name = "exercise"
+
+    def lookups(self, request, model_admin):
+        qs = (
+            ExerciseEmailProperties.objects.filter_by_user(user=request.user)
+            .values_list("exercise", flat=True)
+            .distinct()
+        )
+        exercises = [[ex, ex] for ex in qs]
+        return exercises
```

Comment:
Replace this block with the following please
```python
    def lookups(self, request, model_admin):
        exercises = Exercise.user_objects.filter(
            deleted_at=None,
            organisation=request.user.organization
        )
        return [(e.id, e) for e in exercises]
```

Reply:
sorry scrap the above, use the filter_by_user from the custom manager instead
```python
def lookups(self, request, model_admin):
    exercises = Exercise.user_objects.filter_by_user(request.user)
    return [(e.id, e) for e in exercises]
```
this should work 👍